### ISSUE TITLE
Bad EER using SpeechBrain pretrained models

### ISSUE THREAD

**Issue by aviasd** [speechbrain/speechbrain#1025]
I tried to check the EER of the pretrained models from [HuggingFace](https://huggingface.co/speechbrain):  
[Ecapa-TDNN](https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb)  
[X-Vectors](https://huggingface.co/speechbrain/spkrec-xvect-voxceleb)  

I run the verification task on [VoxCeleb1](https://www.robots.ox.ac.uk/~vgg/data/voxceleb/vox1.html) data with the [Test Clean](https://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/veri_test2.txt) split.  
I used the [inference code](https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb#perform-speaker-verification), with the  `verify_files()` method.  
I used EER from [speechbrain/utils/metric_stats.py](https://github.com/speechbrain/speechbrain/blob/develop/speechbrain/utils/metric_stats.py#L413) with:
```from speechbrain.utils.metric_stats import EER```
I got around **44%** EER, and not **0.69%** EER as you published.  
I know that others had the same issue, I think that maybe the models you uploaded are not the trained models.

**Comment by mravanelli** [comment]
What happens if you use the official script [here](https://github.com/speechbrain/speechbrain/blob/develop/recipes/VoxCeleb/SpeakerRec/speaker_verification_cosine.py) for the evaluation? It uses the models we have on HuggingFace and I can see an EER=0.69 with that.

**Comment by aviasd** [comment]
I ran [speechbrain/recipes/VoxCeleb/SpeakerRec/speaker_verification_cosine.py](https://github.com/speechbrain/speechbrain/blob/develop/recipes/VoxCeleb/SpeakerRec/speaker_verification_cosine.py) script, and it worked well.  
I guess the reason is that in this script, you are doing a lot more than just computing embeddings and using EER on them:
1. Convert WAV to features
2. Normalize features
3. Convert features to embeddings
4. Normalize embeddings

After the end of the cycle, you do the same again (the normalization is more stable the second time).

In my approach (which didn't work), I only generated embeddings and then calculated EER.
